plugins/action/ci_net_map.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+# Copyright Red Hat, Inc.
+# Apache License Version 2.0 (see LICENSE)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import typing
+
+DOCUMENTATION = r"""
+---
+action: ci_net_map
+
+short_description: Maps the Networking Definition to a Networking Env. Definition
+
+description:
+- Takes the Networking Definition, and based on Ansible facts and a set of
+  rules, translates that info into a valid Networking Env. Definition.
+- This task plugin requires networking facts to be present for every 
+  instance referenced in the Networking Definition.
+- Partial mappings are allowed to allow infrastructure bootstrapping based on the
+  mapping result even without already deployed instances.
+
+options:
+  networking_definition:
+    description:
+    - The Networking Definition as a dictionary.
+    type: str
```

Comment:
Does this mean that `networking_definition` should be passed as a string containing a dictionary, or the type should be dict?